Remove dependency to dunce (normalizes Windows paths to the most compatible format)

--- icedtea-web-1.8.8/rust-launcher/Cargo.toml			2021-10-27 22:13:18.000000000 +0200
+++ icedtea-web-1.8.8/rust-launcher/Cargo.toml.remove-dunce	2023-04-23 00:48:27.861178475 +0200
@@ -2,7 +2,3 @@
 name = "launcher"
 version = "1.8.0"
 authors = ["https://icedtea.classpath.org/wiki/IcedTea-Web"]
-
-[dependencies]
-[target.'cfg(windows)'.dependencies]
-dunce = "0.1.1"
